### ISSUE TITLE
Medusa unresponsive when listing show

### ISSUE THREAD

**Issue by gilfrade** [pymedusa/Medusa#5170]
**Describe the bug**
Generate a new install of medusa. Imported one existing show (all OK). Edited imported show (all OK). Imported several Anime shows (all OK). Tried to edit any show and hangs. Only way to recover is to kill process and launch it again. Nothing on logs (with all active) useful at the moment of the hangs.

Tried several times from the start and it seems to hang when a show is in tvdbv2 but then cannot retrieve show information. Example: FLCL Progressive is not available in English on tvdbv2 but it is in Japanese. After this every time a tried to list a show details it hang. 

**To Reproduce**
1. Install Fedora Server 28
2. Git clone official medusa (master)
3. Enter medusa web and add multiple existing shows. Repeat again.
4. See image

**Expected behavior**
View show details

**Screenshots**
![medusa_ws_pending](https://user-images.githubusercontent.com/1220485/45259902-e23c3980-b3cf-11e8-8dc0-e4a402784e12.PNG)

**Medusa (please complete the following information):**
 - OS: Fedora Server 28
 - Branch: master
 - Commit:  db3d9d7

**Logs:**
```
**HANG**
2018-09-09 01:20:14 INFO     TORNADO :: [db3d9d7] 200 GET /fonts/glyphicons-halflings-regular.woff2 (127.0.0.1) 4.94ms
2018-09-09 01:20:14 INFO     TORNADO :: [db3d9d7] 200 GET /images/menu/16x_sprite_colored_menu_icons.png (127.0.0.1) 7.63ms
2018-09-09 01:20:14 INFO     TORNADO :: [db3d9d7] 200 GET /css/lib/images/ui-icons_ffffff_256x240.png (127.0.0.1) 10.34ms
2018-09-09 01:20:13 INFO     TORNADO :: [db3d9d7] 200 GET /js/vendors.js.map (127.0.0.1) 20.72ms
2018-09-09 01:20:13 INFO     TORNADO :: [db3d9d7] 200 GET /js/index.js.map (127.0.0.1) 2.41ms
2018-09-09 01:20:13 INFO     TORNADO :: [db3d9d7] 200 GET /api/v2/config/main (127.0.0.1) 4.98ms
2018-09-09 01:20:13 INFO     TORNADO :: [db3d9d7] 200 GET /css/vendors.css.map (127.0.0.1) 12.65ms
2018-09-09 01:20:13 INFO     TORNADO :: [db3d9d7] 101 GET /ws/ui (127.0.0.1) 0.68ms
2018-09-09 01:20:13 INFO     TORNADO :: [db3d9d7] 200 GET /js/ajax-episode-subtitles.js?10641 (127.0.0.1) 9.91ms
2018-09-09 01:20:13 INFO     TORNADO :: [db3d9d7] 200 GET /js/ajax-episode-search.js?10641 (127.0.0.1) 11.00ms
2018-09-09 01:20:13 INFO     TORNADO :: [db3d9d7] 200 GET /js/rating-tooltip.js?10641 (127.0.0.1) 11.74ms
2018-09-09 01:20:13 INFO     TORNADO :: [db3d9d7] 200 GET /js/notifications.js?10641 (127.0.0.1) 9.63ms
2018-09-09 01:20:13 INFO     TORNADO :: [db3d9d7] 200 GET /js/browser.js?10641 (127.0.0.1) 10.84ms
2018-09-09 01:20:13 INFO     TORNADO :: [db3d9d7] 200 GET /js/manage/subtitle-missed.js?10641 (127.0.0.1) 11.72ms
2018-09-09 01:20:13 INFO     TORNADO :: [db3d9d7] 200 GET /js/manage/init.js?10641 (127.0.0.1) 15.28ms
2018-09-09 01:20:13 INFO     TORNADO :: [db3d9d7] 200 GET /js/manage/index.js?10641 (127.0.0.1) 16.28ms
2018-09-09 01:20:13 INFO     TORNADO :: [db3d9d7] 200 GET /js/vender.min.js?10641 (127.0.0.1) 190.75ms
2018-09-09 01:20:13 INFO     TORNADO :: [db3d9d7] 200 GET /js/manage/failed-downloads.js?10641 (127.0.0.1) 14.13ms
2018-09-09 01:20:13 INFO     TORNADO :: [db3d9d7] 200 GET /js/home/restart.js?10641 (127.0.0.1) 14.95ms
2018-09-09 01:20:13 INFO     TORNADO :: [db3d9d7] 200 GET /js/common/init.js?10641 (127.0.0.1) 16.09ms
2018-09-09 01:20:13 INFO     TORNADO :: [db3d9d7] 200 GET /js/add-shows/trending-shows.js?10641 (127.0.0.1) 19.00ms
2018-09-09 01:20:13 INFO     TORNADO :: [db3d9d7] 200 GET /js/add-shows/recommended-shows.js?10641 (127.0.0.1) 19.84ms
2018-09-09 01:20:13 INFO     TORNADO :: [db3d9d7] 200 GET /js/vendors.js?10641 (127.0.0.1) 174.53ms
2018-09-09 01:20:13 INFO     TORNADO :: [db3d9d7] 200 GET /js/add-shows/popular-shows.js?10641 (127.0.0.1) 19.05ms
2018-09-09 01:20:13 INFO     TORNADO :: [db3d9d7] 200 GET /js/add-shows/init.js?10641 (127.0.0.1) 20.39ms
2018-09-09 01:20:13 INFO     TORNADO :: [db3d9d7] 200 GET /js/config/notifications.js?10641 (127.0.0.1) 23.22ms
2018-09-09 01:20:13 INFO     TORNADO :: [db3d9d7] 200 GET /js/config/init.js?10641 (127.0.0.1) 24.32ms
2018-09-09 01:20:13 INFO     TORNADO :: [db3d9d7] 200 GET /js/config/index.js?10641 (127.0.0.1) 25.05ms
2018-09-09 01:20:13 INFO     TORNADO :: [db3d9d7] 200 GET /js/lib/bootstrap-formhelpers.min.js?10641 (127.0.0.1) 122.19ms
2018-09-09 01:20:13 INFO     TORNADO :: [db3d9d7] 200 GET /js/lib/date_fns.min.js?10641 (127.0.0.1) 49.65ms
2018-09-09 01:20:13 INFO     TORNADO :: [db3d9d7] 200 GET /js/parsers.js?10641 (127.0.0.1) 26.89ms
2018-09-09 01:20:13 INFO     TORNADO :: [db3d9d7] 200 GET /js/lib/lazyload.js?10641 (127.0.0.1) 36.93ms
2018-09-09 01:20:13 INFO     TORNADO :: [db3d9d7] 200 GET /fonts/open-sans-v15-latin-italic.ttf (127.0.0.1) 37.96ms
2018-09-09 01:20:13 INFO     TORNADO :: [db3d9d7] 200 GET /fonts/open-sans-v15-latin-regular.ttf (127.0.0.1) 39.15ms
2018-09-09 01:20:13 INFO     TORNADO :: [db3d9d7] 200 GET /js/lib/formwizard.js?10641 (127.0.0.1) 41.44ms
2018-09-09 01:20:13 INFO     TORNADO :: [db3d9d7] 200 GET /js/lib/fix-broken-ie.js?10641 (127.0.0.1) 29.72ms
2018-09-09 01:20:13 INFO     TORNADO :: [db3d9d7] 200 GET /js/index.js?10641 (127.0.0.1) 42.59ms
2018-09-09 01:20:13 INFO     TORNADO :: [db3d9d7] 200 GET /css/style.css?10641 (127.0.0.1) 54.62ms
2018-09-09 01:20:13 INFO     TORNADO :: [db3d9d7] 200 GET /css/lib/vue-snotify-material.css?10641 (127.0.0.1) 19.67ms
2018-09-09 01:20:13 INFO     TORNADO :: [db3d9d7] 200 GET /css/country-flags.css?10641 (127.0.0.1) 20.77ms
2018-09-09 01:20:13 INFO     TORNADO :: [db3d9d7] 200 GET /css/print.css?10641 (127.0.0.1) 21.58ms
2018-09-09 01:20:13 INFO     TORNADO :: [db3d9d7] 200 GET /css/themed.css?10641 (127.0.0.1) 23.78ms
2018-09-09 01:20:13 INFO     TORNADO :: [db3d9d7] 200 GET /css/vendors.css?10641 (127.0.0.1) 41.66ms
2018-09-09 01:20:13 INFO     TORNADO :: [db3d9d7] 200 GET /css/lib/jquery.qtip-2.2.1.min.css?10641 (127.0.0.1) 8.71ms
2018-09-09 01:20:13 INFO     TORNADO :: [db3d9d7] 200 GET /css/lib/jquery-ui-1.10.4.custom.min.css?10641 (127.0.0.1) 10.48ms
2018-09-09 01:20:13 INFO     TORNADO :: [db3d9d7] 200 GET /css/browser.css?10641 (127.0.0.1) 11.40ms
2018-09-09 01:20:13 INFO     TORNADO :: [db3d9d7] 200 GET /css/bootstrap-formhelpers.min.css?10641 (127.0.0.1) 14.22ms
2018-09-09 01:20:13 INFO     TORNADO :: [db3d9d7] 200 GET /css/vender.min.css?10641 (127.0.0.1) 16.05ms
2018-09-09 01:20:13 INFO     TORNADO :: [db3d9d7] 200 GET /home/displayShow?indexername=tvdb&seriesid=317953 (127.0.0.1) 783.12ms
2018-09-09 01:20:10 INFO     TORNADO :: [db3d9d7] 200 GET /api/v2/series/tvdb267440/asset/fanart?api_key=**********
2018-09-09 01:20:10 INFO     TORNADO :: [db3d9d7] 200 GET /images/menu/system18-2.png (127.0.0.1) 2.80ms
2018-09-09 01:20:10 INFO     TORNADO :: [db3d9d7] 200 GET /images/menu/system18.png (127.0.0.1) 4.06ms
2018-09-09 01:20:10 INFO     TORNADO :: [db3d9d7] 200 GET /api/v2/series/tvdb294002/asset/network?api_key=**********
2018-09-09 01:20:10 INFO     TORNADO :: [db3d9d7] 200 GET /api/v2/series/tvdb294002/asset/posterThumb?api_key=**********
2018-09-09 01:20:10 INFO     TORNADO :: [db3d9d7] 200 GET /api/v2/series/tvdb305074/asset/network?api_key=**********
2018-09-09 01:20:10 INFO     TORNADO :: [db3d9d7] 200 GET /api/v2/series/tvdb305074/asset/posterThumb?api_key=**********
2018-09-09 01:20:10 INFO     TORNADO :: [db3d9d7] 200 GET /api/v2/series/tvdb317953/asset/posterThumb?api_key=**********
2018-09-09 01:20:10 INFO     TORNADO :: [db3d9d7] 200 GET /api/v2/series/tvdb317953/asset/network?api_key=**********
2018-09-09 01:20:10 INFO     TORNADO :: [db3d9d7] 200 GET /images/xem.png (127.0.0.1) 34.22ms
2018-09-09 01:20:10 INFO     TORNADO :: [db3d9d7] 200 GET /images/poster-back-dark.png (127.0.0.1) 37.83ms
2018-09-09 01:20:10 INFO     TORNADO :: [db3d9d7] 200 GET /api/v2/series/tvdb326109/asset/network?api_key=**********
2018-09-09 01:20:10 INFO     TORNADO :: [db3d9d7] 200 GET /api/v2/series/tvdb326109/asset/posterThumb?api_key=**********
2018-09-09 01:20:10 INFO     TORNADO :: [db3d9d7] 200 GET /api/v2/series/tvdb331787/asset/network?api_key=**********
2018-09-09 01:20:10 INFO     TORNADO :: [db3d9d7] 200 GET /api/v2/series/tvdb331787/asset/posterThumb?api_key=**********
2018-09-09 01:20:10 INFO     TORNADO :: [db3d9d7] 200 GET /api/v2/series/tvdb334180/asset/network?api_key=**********
2018-09-09 01:20:10 INFO     TORNADO :: [db3d9d7] 200 GET /api/v2/series/tvdb334180/asset/posterThumb?api_key=**********
2018-09-09 01:20:10 INFO     TORNADO :: [db3d9d7] 200 GET /api/v2/config/ (127.0.0.1) 4.31ms
2018-09-09 01:20:10 INFO     TORNADO :: [db3d9d7] 200 GET /api/v2/series/tvdb331753/asset/network?api_key=**********
2018-09-09 01:20:10 INFO     TORNADO :: [db3d9d7] 200 GET /api/v2/series/tvdb331753/asset/posterThumb?api_key=**********
2018-09-09 01:20:10 INFO     TORNADO :: [db3d9d7] 200 GET /api/v2/series/tvdb267440/asset/network?api_key=**********
2018-09-09 01:20:10 INFO     TORNADO :: [db3d9d7] 200 GET /api/v2/series/tvdb267440/asset/posterThumb?api_key=**********
2018-09-09 01:20:10 INFO     TORNADO :: [db3d9d7] 200 GET /images/medusa.png (127.0.0.1) 2.02ms
2018-09-09 01:20:10 INFO     TORNADO :: [db3d9d7] 200 GET /fonts/open-sans-v15-latin-700.ttf (127.0.0.1) 3.38ms
2018-09-09 01:20:10 INFO     TORNADO :: [db3d9d7] 200 GET /fonts/glyphicons-halflings-regular.woff2 (127.0.0.1) 3.54ms
2018-09-09 01:20:10 INFO     TORNADO :: [db3d9d7] 200 GET /images/loading32-dark.gif (127.0.0.1) 5.02ms
2018-09-09 01:20:10 INFO     TORNADO :: [db3d9d7] 200 GET /js/vendors.js.map (127.0.0.1) 20.69ms
2018-09-09 01:20:10 INFO     TORNADO :: [db3d9d7] 200 GET /js/index.js.map (127.0.0.1) 1.83ms
2018-09-09 01:20:10 INFO     TORNADO :: [db3d9d7] 200 GET /api/v2/config/main (127.0.0.1) 5.14ms
2018-09-09 01:20:10 INFO     TORNADO :: [db3d9d7] 101 GET /ws/ui (127.0.0.1) 0.72ms
2018-09-09 01:20:10 INFO     TORNADO :: [db3d9d7] 200 GET /css/vendors.css.map (127.0.0.1) 10.66ms
2018-09-09 01:20:10 INFO     TORNADO :: [db3d9d7] 200 GET /js/notifications.js?10641 (127.0.0.1) 11.24ms
2018-09-09 01:20:10 INFO     TORNADO :: [db3d9d7] 200 GET /js/browser.js?10641 (127.0.0.1) 12.50ms
2018-09-09 01:20:10 INFO     TORNADO :: [db3d9d7] 200 GET /js/manage/subtitle-missed.js?10641 (127.0.0.1) 13.57ms
2018-09-09 01:20:10 INFO     TORNADO :: [db3d9d7] 200 GET /js/vender.min.js?10641 (127.0.0.1) 191.23ms
2018-09-09 01:20:10 INFO     TORNADO :: [db3d9d7] 200 GET /js/manage/init.js?10641 (127.0.0.1) 16.15ms
2018-09-09 01:20:10 INFO     TORNADO :: [db3d9d7] 200 GET /js/manage/index.js?10641 (127.0.0.1) 17.08ms
2018-09-09 01:20:10 INFO     TORNADO :: [db3d9d7] 200 GET /js/manage/failed-downloads.js?10641 (127.0.0.1) 13.83ms
2018-09-09 01:20:10 INFO     TORNADO :: [db3d9d7] 200 GET /js/home/restart.js?10641 (127.0.0.1) 14.79ms
2018-09-09 01:20:10 INFO     TORNADO :: [db3d9d7] 200 GET /js/common/init.js?10641 (127.0.0.1) 15.98ms
2018-09-09 01:20:10 INFO     TORNADO :: [db3d9d7] 200 GET /js/add-shows/trending-shows.js?10641 (127.0.0.1) 24.13ms
2018-09-09 01:20:10 INFO     TORNADO :: [db3d9d7] 200 GET /js/add-shows/recommended-shows.js?10641 (127.0.0.1) 25.00ms
2018-09-09 01:20:10 INFO     TORNADO :: [db3d9d7] 200 GET /js/vendors.js?10641 (127.0.0.1) 170.81ms
2018-09-09 01:20:10 INFO     TORNADO :: [db3d9d7] 200 GET /js/add-shows/popular-shows.js?10641 (127.0.0.1) 18.39ms
2018-09-09 01:20:10 INFO     TORNADO :: [db3d9d7] 200 GET /js/add-shows/init.js?10641 (127.0.0.1) 19.58ms
2018-09-09 01:20:10 INFO     TORNADO :: [db3d9d7] 200 GET /js/config/notifications.js?10641 (127.0.0.1) 22.36ms
2018-09-09 01:20:10 INFO     TORNADO :: [db3d9d7] 200 GET /js/config/init.js?10641 (127.0.0.1) 23.45ms
2018-09-09 01:20:10 INFO     TORNADO :: [db3d9d7] 200 GET /js/config/index.js?10641 (127.0.0.1) 24.10ms
2018-09-09 01:20:10 INFO     TORNADO :: [db3d9d7] 200 GET /js/lib/bootstrap-formhelpers.min.js?10641 (127.0.0.1) 120.44ms
2018-09-09 01:20:10 INFO     TORNADO :: [db3d9d7] 200 GET /js/lib/date_fns.min.js?10641 (127.0.0.1) 48.55ms
2018-09-09 01:20:10 INFO     TORNADO :: [db3d9d7] 200 GET /js/parsers.js?10641 (127.0.0.1) 26.30ms
2018-09-09 01:20:10 INFO     TORNADO :: [db3d9d7] 200 GET /js/lib/lazyload.js?10641 (127.0.0.1) 36.53ms
2018-09-09 01:20:10 INFO     TORNADO :: [db3d9d7] 200 GET /fonts/open-sans-v15-latin-italic.ttf (127.0.0.1) 37.47ms
2018-09-09 01:20:10 INFO     TORNADO :: [db3d9d7] 200 GET /fonts/open-sans-v15-latin-regular.ttf (127.0.0.1) 38.61ms
2018-09-09 01:20:10 INFO     TORNADO :: [db3d9d7] 200 GET /js/lib/formwizard.js?10641 (127.0.0.1) 40.74ms
2018-09-09 01:20:10 INFO     TORNADO :: [db3d9d7] 200 GET /js/lib/fix-broken-ie.js?10641 (127.0.0.1) 29.92ms
2018-09-09 01:20:10 INFO     TORNADO :: [db3d9d7] 200 GET /js/index.js?10641 (127.0.0.1) 42.71ms
2018-09-09 01:20:10 INFO     TORNADO :: [db3d9d7] 200 GET /css/style.css?10641 (127.0.0.1) 53.37ms
2018-09-09 01:20:10 INFO     TORNADO :: [db3d9d7] 200 GET /css/lib/vue-snotify-material.css?10641 (127.0.0.1) 18.71ms
2018-09-09 01:20:10 INFO     TORNADO :: [db3d9d7] 200 GET /css/country-flags.css?10641 (127.0.0.1) 19.90ms
2018-09-09 01:20:10 INFO     TORNADO :: [db3d9d7] 200 GET /css/print.css?10641 (127.0.0.1) 20.76ms
2018-09-09 01:20:10 INFO     TORNADO :: [db3d9d7] 200 GET /css/themed.css?10641 (127.0.0.1) 22.96ms
2018-09-09 01:20:10 INFO     TORNADO :: [db3d9d7] 200 GET /css/vendors.css?10641 (127.0.0.1) 42.84ms
2018-09-09 01:20:10 INFO     TORNADO :: [db3d9d7] 200 GET /css/lib/jquery.qtip-2.2.1.min.css?10641 (127.0.0.1) 8.70ms
2018-09-09 01:20:10 INFO     TORNADO :: [db3d9d7] 200 GET /css/lib/jquery-ui-1.10.4.custom.min.css?10641 (127.0.0.1) 10.43ms
2018-09-09 01:20:10 INFO     TORNADO :: [db3d9d7] 200 GET /css/browser.css?10641 (127.0.0.1) 11.32ms
2018-09-09 01:20:10 INFO     TORNADO :: [db3d9d7] 200 GET /css/bootstrap-formhelpers.min.css?10641 (127.0.0.1) 13.50ms
2018-09-09 01:20:10 INFO     TORNADO :: [db3d9d7] 200 GET /css/vender.min.css?10641 (127.0.0.1) 15.25ms
2018-09-09 01:20:10 INFO     TORNADO :: [db3d9d7] 200 GET /home/ (127.0.0.1) 111.71ms
2018-09-09 01:20:08 INFO     TORNADO :: [db3d9d7] 200 GET /images/menu/16x_sprite_colored_menu_icons.png (127.0.0.1) 3.50ms
2018-09-09 01:20:07 INFO     TORNADO :: [db3d9d7] 200 GET /images/banner.png (127.0.0.1) 2.84ms
2018-09-09 01:20:07 INFO     TORNADO :: [db3d9d7] 200 GET /home/getManualSearchStatus (127.0.0.1) 1.78ms
2018-09-09 01:20:07 INFO     TORNADO :: [db3d9d7] 200 GET /api/v2/series/tvdb334180/asset/fanart?api_key=**********
2018-09-09 01:20:07 INFO     TORNADO :: [db3d9d7] 200 GET /images/menu/system18-2.png (127.0.0.1) 4.26ms
2018-09-09 01:20:07 INFO     TORNADO :: [db3d9d7] 200 GET /images/menu/system18.png (127.0.0.1) 5.36ms
2018-09-09 01:20:07 INFO     TORNADO :: [db3d9d7] 404 GET /api/v2/series/tvdb334180/asset/banner?api_key=**********
2018-09-09 01:20:07 INFO     TORNADO :: [db3d9d7] 200 GET /api/v2/series/tvdb305074/asset/banner?api_key=**********
2018-09-09 01:20:07 INFO     TORNADO :: [db3d9d7] 200 GET /api/v2/config/ (127.0.0.1) 5.26ms
2018-09-09 01:20:07 INFO     TORNADO :: [db3d9d7] 200 GET /api/v2/series/tvdb294002/asset/banner?api_key=**********
2018-09-09 01:20:07 INFO     TORNADO :: [db3d9d7] 200 GET /images/medusa.png (127.0.0.1) 5.73ms
2018-09-09 01:20:07 INFO     TORNADO :: [db3d9d7] 200 GET /api/v2/series/tvdb267440/asset/banner?api_key=**********
2018-09-09 01:20:06 INFO     TORNADO :: [db3d9d7] 200 GET /fonts/glyphicons-halflings-regular.woff2 (127.0.0.1) 4.50ms
2018-09-09 01:20:06 INFO     TORNADO :: [db3d9d7] 200 GET /fonts/open-sans-v15-latin-700.ttf (127.0.0.1) 7.41ms
2018-09-09 01:20:06 INFO     TORNADO :: [db3d9d7] 200 GET /images/glyphicons-halflings-white.png (127.0.0.1) 10.05ms
2018-09-09 01:20:06 INFO     TORNADO :: [db3d9d7] 200 GET /js/vendors.js.map (127.0.0.1) 23.97ms
2018-09-09 01:20:06 INFO     TORNADO :: [db3d9d7] 200 GET /js/index.js.map (127.0.0.1) 2.30ms
2018-09-09 01:20:06 INFO     TORNADO :: [db3d9d7] 200 GET /api/v2/config/main (127.0.0.1) 7.74ms
2018-09-09 01:20:06 INFO     TORNADO :: [db3d9d7] 101 GET /ws/ui (127.0.0.1) 0.91ms
2018-09-09 01:20:06 INFO     TORNADO :: [db3d9d7] 200 GET /images/plus.png (127.0.0.1) 5.15ms
2018-09-09 01:20:06 INFO     TORNADO :: [db3d9d7] 200 GET /images/manualsearch.png (127.0.0.1) 6.17ms
2018-09-09 01:20:06 INFO     TORNADO :: [db3d9d7] 200 GET /images/search16.png (127.0.0.1) 9.40ms
2018-09-09 01:20:06 INFO     TORNADO :: [db3d9d7] 200 GET /js/ajax-episode-search.js?10641 (127.0.0.1) 11.72ms
2018-09-09 01:20:06 INFO     TORNADO :: [db3d9d7] 200 GET /js/notifications.js?10641 (127.0.0.1) 12.77ms
2018-09-09 01:20:06 INFO     TORNADO :: [db3d9d7] 200 GET /js/browser.js?10641 (127.0.0.1) 14.06ms
2018-09-09 01:20:06 INFO     TORNADO :: [db3d9d7] 200 GET /js/manage/subtitle-missed.js?10641 (127.0.0.1) 14.96ms
2018-09-09 01:20:06 INFO     TORNADO :: [db3d9d7] 200 GET /js/manage/init.js?10641 (127.0.0.1) 17.92ms
2018-09-09 01:20:06 INFO     TORNADO :: [db3d9d7] 200 GET /js/manage/index.js?10641 (127.0.0.1) 11.04ms
2018-09-09 01:20:06 INFO     TORNADO :: [db3d9d7] 200 GET /js/manage/failed-downloads.js?10641 (127.0.0.1) 12.10ms
2018-09-09 01:20:06 INFO     TORNADO :: [db3d9d7] 200 GET /js/vender.min.js?10641 (127.0.0.1) 191.96ms
2018-09-09 01:20:06 INFO     TORNADO :: [db3d9d7] 200 GET /js/home/restart.js?10641 (127.0.0.1) 14.20ms
2018-09-09 01:20:06 INFO     TORNADO :: [db3d9d7] 200 GET /js/common/init.js?10641 (127.0.0.1) 15.38ms
2018-09-09 01:20:06 INFO     TORNADO :: [db3d9d7] 200 GET /js/add-shows/trending-shows.js?10641 (127.0.0.1) 16.26ms
2018-09-09 01:20:06 INFO     TORNADO :: [db3d9d7] 200 GET /js/add-shows/recommended-shows.js?10641 (127.0.0.1) 20.13ms
2018-09-09 01:20:06 INFO     TORNADO :: [db3d9d7] 200 GET /js/vendors.js?10641 (127.0.0.1) 186.59ms
2018-09-09 01:20:06 INFO     TORNADO :: [db3d9d7] 200 GET /js/add-shows/popular-shows.js?10641 (127.0.0.1) 18.40ms
2018-09-09 01:20:06 INFO     TORNADO :: [db3d9d7] 200 GET /js/add-shows/init.js?10641 (127.0.0.1) 28.31ms
2018-09-09 01:20:06 INFO     TORNADO :: [db3d9d7] 200 GET /js/config/notifications.js?10641 (127.0.0.1) 30.41ms
2018-09-09 01:20:06 INFO     TORNADO :: [db3d9d7] 200 GET /css/vendors.css.map (127.0.0.1) 63.70ms
2018-09-09 01:20:06 INFO     TORNADO :: [db3d9d7] 200 GET /js/config/init.js?10641 (127.0.0.1) 22.51ms
2018-09-09 01:20:06 INFO     TORNADO :: [db3d9d7] 200 GET /js/lib/bootstrap-formhelpers.min.js?10641 (127.0.0.1) 120.59ms
2018-09-09 01:20:06 INFO     TORNADO :: [db3d9d7] 200 GET /js/config/index.js?10641 (127.0.0.1) 29.30ms
2018-09-09 01:20:06 INFO     TORNADO :: [db3d9d7] 200 GET /js/lib/date_fns.min.js?10641 (127.0.0.1) 48.60ms
2018-09-09 01:20:06 INFO     TORNADO :: [db3d9d7] 200 GET /js/parsers.js?10641 (127.0.0.1) 25.15ms
2018-09-09 01:20:06 INFO     TORNADO :: [db3d9d7] 200 GET /js/lib/lazyload.js?10641 (127.0.0.1) 48.41ms
2018-09-09 01:20:06 INFO     TORNADO :: [db3d9d7] 200 GET /fonts/open-sans-v15-latin-italic.ttf (127.0.0.1) 49.36ms
2018-09-09 01:20:06 INFO     TORNADO :: [db3d9d7] 200 GET /fonts/open-sans-v15-latin-regular.ttf (127.0.0.1) 50.54ms
2018-09-09 01:20:06 INFO     TORNADO :: [db3d9d7] 200 GET /js/lib/formwizard.js?10641 (127.0.0.1) 24.70ms
2018-09-09 01:20:06 INFO     TORNADO :: [db3d9d7] 200 GET /js/lib/fix-broken-ie.js?10641 (127.0.0.1) 25.50ms
2018-09-09 01:20:06 INFO     TORNADO :: [db3d9d7] 200 GET /js/index.js?10641 (127.0.0.1) 39.30ms
2018-09-09 01:20:06 INFO     TORNADO :: [db3d9d7] 200 GET /images/thetvdb16.png (127.0.0.1) 33.93ms
2018-09-09 01:20:06 INFO     TORNADO :: [db3d9d7] 200 GET /images/imdb.png (127.0.0.1) 35.30ms
2018-09-09 01:20:06 INFO     TORNADO :: [db3d9d7] 200 GET /css/style.css?10641 (127.0.0.1) 29.88ms
2018-09-09 01:20:06 INFO     TORNADO :: [db3d9d7] 200 GET /css/lib/vue-snotify-material.css?10641 (127.0.0.1) 11.06ms
2018-09-09 01:20:06 INFO     TORNADO :: [db3d9d7] 200 GET /css/country-flags.css?10641 (127.0.0.1) 12.33ms
2018-09-09 01:20:06 INFO     TORNADO :: [db3d9d7] 200 GET /css/print.css?10641 (127.0.0.1) 13.25ms
2018-09-09 01:20:06 INFO     TORNADO :: [db3d9d7] 200 GET /css/themed.css?10641 (127.0.0.1) 15.51ms
2018-09-09 01:20:06 INFO     TORNADO :: [db3d9d7] 200 GET /css/vendors.css?10641 (127.0.0.1) 46.29ms
2018-09-09 01:20:06 INFO     TORNADO :: [db3d9d7] 200 GET /css/lib/jquery.qtip-2.2.1.min.css?10641 (127.0.0.1) 9.57ms
2018-09-09 01:20:06 INFO     TORNADO :: [db3d9d7] 200 GET /css/lib/jquery-ui-1.10.4.custom.min.css?10641 (127.0.0.1) 11.54ms
2018-09-09 01:20:06 INFO     TORNADO :: [db3d9d7] 200 GET /css/browser.css?10641 (127.0.0.1) 12.85ms
2018-09-09 01:20:06 INFO     TORNADO :: [db3d9d7] 200 GET /css/bootstrap-formhelpers.min.css?10641 (127.0.0.1) 14.70ms
2018-09-09 01:20:06 INFO     TORNADO :: [db3d9d7] 200 GET /css/vender.min.css?10641 (127.0.0.1) 16.73ms
2018-09-09 01:20:06 INFO     TORNADO :: [db3d9d7] 200 GET /schedule/ (127.0.0.1) 341.44ms
2018-09-09 01:20:06 INFO     TORNADO :: [db3d9d7] 302 GET / (127.0.0.1) 2.50ms
2018-09-09 01:20:05 INFO     TORNADO :: [db3d9d7] 200 GET /home/getManualSearchStatus?indexername=tvdb&seriesid=331753 (127.0.0.1) 5.45ms
2018-09-09 01:20:02 INFO     CHECKVERSION :: [db3d9d7] Checking for updates using GIT
2018-09-09 01:20:00 INFO     MAIN :: [db3d9d7] Building internal name cache for all shows
2018-09-09 01:20:00 INFO     MAIN :: [db3d9d7] Finished processing 2850 scene exceptions.
2018-09-09 01:20:00 INFO     MAIN :: [db3d9d7] Updating exception_cache and exception_season_cache
2018-09-09 01:20:00 INFO     TORNADO :: [db3d9d7] Starting Medusa on http://0.0.0.0:8081/
2018-09-09 01:20:00 INFO     MAIN :: [db3d9d7] Starting Medusa [master] using u'/opt/src/Medusa/config.ini'
2018-09-09 01:20:00 INFO     MAIN :: [db3d9d7] Broken providers found: [u'torrent9', u'newpct']
```
</details>

**Additional context**
Assumed to problem was something on websocket since its always the first to became "pending". Added nginx as reverse proxy just to check its behavior and its the same.

**Comment by p0psicles** [comment]
Its the anidb issue. We're working on it

**Comment by p0psicles** [comment]
Temporary disable anidb release group lookup, fixes the issue for now

**Comment by gilfrade** [comment]
I do not think thats the case, one of the tries was without "anime" feature active and it still hang.

**Comment by OmgImAlexis** [comment]
@gilfrade there's a very good chance @p0psicles is right here. The `series` endpoint you're seeing a 404 on gets all the shows, that's related to the bug @p0psicles mentioned.

**Comment by sharkykh** [comment]
I agree this is the bug with calling AniDB for all the shows. Fixed with #5166

@gilfrade As for your question: the request is canceled client side only after a timeout of 10 seconds. The request handling on the tornado server is still continuing.
It hangs the whole server because the API endpoint is not multi-threaded yet, but it was fixed by #4979 which we will include in 0.2.11 (**not** the upcoming hotfix)
